docker-compose.yml

```diff
@@ -20,11 +33,9 @@ services:
     build:
       context: ui
       dockerfile: Dockerfile
-    volumes:
-      - /frontend:/app
     network_mode: "host"
 
-  postgresql-cue:
+  cueo-postgresql:
```

Comment:
Postgres is not needed in docker-compose, sqllite will be used in that scenario.

Reply:
As discussed, let's just keep this in docker-compose-dev and remove it from docker-compose (later)